plugin/src/Theme.lua

```diff
@@ -11,10 +11,34 @@ local Theme = {
 	LightTextColor = Color3.fromRGB(160, 160, 160),
 }
 
-setmetatable(Theme, {
+local DarkTheme = {
+
+	-- Little unsure on how to change the color of the background on the plugin for dark mode...
```

Comment:
We'll probably need to put the remaining colors into our theme tables!

Reply:
Yeah, we should.